mx.sulong/mx_sulong.py

```diff
@@ -56,6 +56,9 @@ def executeGate():
         with Task('TestPolglot', tasks) as t:
```

Comment:
@woess I think that an entry in the `.gitignore` file is sufficient for now: #179.